package.json

```diff
@@ -27,20 +27,27 @@
   "author": "rt2zz <zack@root-two.com>",
   "license": "MIT",
   "dependencies": {
-    "lodash.foreach": "^3.0.3",
-    "lodash.isplainobject": "^3.2.0"
+    "lodash.foreach": "^4.1.1",
+    "lodash.isplainobject": "^4.0.3"
   },
   "devDependencies": {
-    "babel": "^5.8.29",
-    "babel-core": "^5.8.33",
-    "babel-eslint": "^4.1.3",
-    "expect": "~1.12.2",
+    "babel-cli": "^6.6.5",
+    "babel-core": "^6.7.4",
+    "babel-eslint": "^6.0.0",
+    "babel-plugin-transform-runtime": "^6.6.0",
+    "babel-preset-es2015": "^6.6.0",
+    "babel-preset-es2015-loose": "^7.0.0",
```

Comment:
What does loose do? If we need to change some code to be more "by the book" I am open to that.